inspirehep/modules/references/view_utils.py

```diff
@@ -85,7 +82,8 @@ def references(self):
                 else:
                     # Easier to use record macros over reference records.
                     if 'publication_info' in reference:
-                        reference['publication_info'] = [reference['publication_info']]
+                        reference['publication_info'] = [
+                            reference['publication_info']]
                     else:
                         reference['publication_info'] = []
```

Comment:
You made it here https://github.com/inspirehep/inspire-next/pull/1409 , that one is still pending as it might also be affected by this PR.